internal/pkg/manifest/job.go

```diff
@@ -93,24 +94,15 @@ func newDefaultScheduledJob() *ScheduledJob {
 func NewScheduledJob(props *ScheduledJobProps) *ScheduledJob {
 	job := newDefaultScheduledJob()
 	// Apply overrides.
-	if props.Name != "" {
-		job.Name = aws.String(props.Name)
-	}
-	if props.Dockerfile != "" {
-		job.ScheduledJobConfig.ImageConfig.Build.BuildArgs.Dockerfile = aws.String(props.Dockerfile)
-	}
-	if props.Image != "" {
-		job.ScheduledJobConfig.ImageConfig.Location = aws.String(props.Image)
-	}
-	if props.Schedule != "" {
-		job.On.Schedule = aws.String(props.Schedule)
-	}
+	job.Name = aws.String(props.Name)
```

Comment:
Should we use `stringP` here as well? Even though `props.Name` won't be `""` with current implementation (right?), it feels to me that we should make as little assumption on the other parts of the codebase as possible.

Reply:
ok that makes sense. I'll update for the other workload types  as well.